src/blocks/IFrameBlock.tsx

```diff
@@ -1,108 +1,161 @@
-import Block from './Block'
```

Comment:
Not sure the intention here with the WebsiteBlock -- we already have the link miniapp, so what's the value of updating this old one here?